### PR TITLE
Fix:ClientSummary&ProfilePas

### PR DESCRIPTION
**Pull request by sansito84**
Fixed some issues at Moto, AP and Home ClientSummary files (with a registered user)
Fixed issue at ProfilePas, do not show PAS modal in admin dashboard
Fixed overflow at modal

### CODE REVIEW

File: src/Pages/Products/Home/ClientSummary/ClientSummary.jsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ export default function ClientSummary({ values, reloadPage }) {
         console.log(values.description.medidas_contra_fuego);
```
Comment (Patriciopg02) [inline src/Pages/Products/Home/ClientSummary/ClientSummary.jsx]: can you delete this console.log?

File: src/components/modal/index.jsx [hunk 2]
```diff
@@ -34,8 +34,8 @@ const handleClick = () => {
 };
 
   return (
-    <div className='modal'>
-        <div className='modal-contenido' ref={menuRef}>
+    <div className='modal ' >
```
Comment (Patriciopg02) [inline src/components/modal/index.jsx]:
```suggestion
    <div className='modal'>
```

Comment (sansito84) [reply src/components/modal/index.jsx]: Done already

File: src/Pages/Products/Home/ClientSummary/ClientSummary.jsx (hunk 1 above)
Comment (sansito84) [reply src/Pages/Products/Home/ClientSummary/ClientSummary.jsx]: Done already

**Review by Patriciopg02 — APPROVED** [review]

**Pull request merged**